lib/unittest.bzl

```diff
@@ -193,20 +193,18 @@ def _make_analysis_test(
     if expect_failure:
         changed_settings["//command_line_option:allow_analysis_failures"] = "True"
 
+    target_attr_kwargs = {}
     if changed_settings:
         test_transition = analysis_test_transition(
             settings = changed_settings,
         )
-        attrs["target_under_test"] = attr.label(
-            aspects = [_action_retrieving_aspect],
-            cfg = test_transition,
-            mandatory = True,
-        )
-    else:
-        attrs["target_under_test"] = attr.label(
-            aspects = [_action_retrieving_aspect],
-            mandatory = True,
-        )
+        target_attr_kwargs["cfg"] = test_transition
+
+    attrs["target_under_test"] = attr.label(
+        aspects = [_action_retrieving_aspect],
+        mandatory = True,
+        **target_attr_kwargs,
```

Comment:
whoops. there's a syntax error here. See bazelCI. Maybe you cannot follow kwargs with a comma?

Reply:
Probably. I was wondering if that's a bug since it's inconsistent with Python's behavior, but turns out it's only inconsistent with _PY3_'s behavior. Python 2 allows a comma after the final argument to a function with the exception of *args or **kwargs. (Python 3 also allows multiple sequences and/or dicts to be unpacked when calling a function.)